src/commands/build.parser.ts

```diff
@@ -64,8 +64,17 @@ If an extension is specified on -o option, this field will be inferenced automat
 preset: A5, A4, A3, B5, B4, JIS-B5, JIS-B4, letter, legal, ledger
 custom(comma separated): 182mm,257mm or 8.5in,11in`,
     )
-    .option('--style <stylesheet>', 'additional stylesheet URL or path')
-    .option('--user-style <user_stylesheet>', 'user stylesheet URL or path')
+    .option('--crop-marks', 'print crop marks')
+    .option('--bleed <bleed>', 'bleed [3mm]')
+    .option('--crop-offset <offset>', 'crop offset [auto]')
```

Comment:
They seem to be very convenient options but I felt the description is not enough. Could you add more description about "bleed" or "crop offset"?